mathesar_ui/src/stores/abstract-types/operations/summarization.ts

```diff
@@ -56,6 +56,14 @@ const functionsResponse: AbstractTypeSummarizationFunctionsResponse = {
       [abstractTypeCategory.Boolean]: abstractTypeCategory.Number,
     },
   },
+  max: {
+    label: 'Max',
+    inputOutputTypeMap: mapInputTypesToTheSameOutputType(),
+  },
+  min: {
+    label: 'Max',
+    inputOutputTypeMap: mapInputTypesToTheSameOutputType(),
+  },
```

Comment:
Please change the label of the `min` function from `'Max'` to `'Min'`.

Reply:
Sorry @seancolsen. Missed this while rebasing. PTAL!